apps/hexagon_benchmarks/median3x3_generator.cpp

```diff
@@ -0,0 +1,78 @@
+#include "Halide.h"
+
+using namespace Halide;
+
+class Median3x3 : public Generator<Median3x3> {
+private:
+    static Expr max3(Expr a, Expr b, Expr c) {
```

Comment:
After #1891 was merged, we can get rid of the min3/max3 functions. Also consider renaming mid3 to just 'mid' after that change.